<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadsControl</class>
 <widget class="QWidget" name="DownloadsControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>28</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>221</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="ctrlFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="MouseHoverButton" name="btnStart">
        <property name="toolTip">
         <string>Start download</string>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/downloadscontrol/play_default</normaloff>
          <normalon>:/downloadscontrol/play_clicked</normalon>
          <disabledoff>:/downloadscontrol/play_disabled</disabledoff>
          <disabledon>:/downloadscontrol/play_disabled</disabledon>
          <activeoff>:/downloadscontrol/play_hover</activeoff>
          <activeon>:/downloadscontrol/play_hover</activeon>
          <selectedoff>:/downloadscontrol/play_default</selectedoff>
          <selectedon>:/downloadscontrol/play_default</selectedon>:/downloadscontrol/play_default</iconset>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MouseHoverButton" name="btnPause">
        <property name="toolTip">
         <string>Pause download</string>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/downloadscontrol/pause_default</normaloff>
          <normalon>:/downloadscontrol/pause_clicked</normalon>
          <disabledoff>:/downloadscontrol/pause_disabled</disabledoff>
          <disabledon>:/downloadscontrol/pause_disabled</disabledon>
          <activeoff>:/downloadscontrol/pause_hover</activeoff>
          <activeon>:/downloadscontrol/pause_hover</activeon>
          <selectedoff>:/downloadscontrol/pause_default</selectedoff>
          <selectedon>:/downloadscontrol/pause_default</selectedon>:/downloadscontrol/pause_default</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MouseHoverButton" name="btnStop">
        <property name="toolTip">
         <string>Stop download</string>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/downloadscontrol/stop_default</normaloff>
          <normalon>:/downloadscontrol/stop_clicked</normalon>
          <disabledoff>:/downloadscontrol/stop_disabled</disabledoff>
          <disabledon>:/downloadscontrol/stop_disabled</disabledon>
          <activeoff>:/downloadscontrol/stop_hover</activeoff>
          <activeon>:/downloadscontrol/stop_hover</activeon>
          <selectedoff>:/downloadscontrol/stop_default</selectedoff>
          <selectedon>:/downloadscontrol/stop_default</selectedon>:/downloadscontrol/stop_default</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MouseHoverButton" name="btnRemove">
        <property name="toolTip">
         <string>Remove download</string>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/downloadscontrol/delete_default</normaloff>
          <normalon>:/downloadscontrol/delete_clicked</normalon>
          <disabledoff>:/downloadscontrol/delete_disabled</disabledoff>
          <disabledon>:/downloadscontrol/delete_disabled</disabledon>
          <activeoff>:/downloadscontrol/delete_hover</activeoff>
          <activeon>:/downloadscontrol/delete_hover</activeon>
          <selectedoff>:/downloadscontrol/delete_default</selectedoff>
          <selectedon>:/downloadscontrol/delete_default</selectedon>:/downloadscontrol/delete_default</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="line">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>1</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MouseHoverButton" name="btnRestart">
        <property name="toolTip">
         <string>Restart download</string>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/downloadscontrol/reload_default</normaloff>
          <normalon>:/downloadscontrol/reload_clicked</normalon>
          <disabledoff>:/downloadscontrol/reload_disabled</disabledoff>
          <disabledon>:/downloadscontrol/reload_disabled</disabledon>
          <activeoff>:/downloadscontrol/reload_hover</activeoff>
          <activeon>:/downloadscontrol/reload_hover</activeon>
          <selectedoff>:/downloadscontrol/reload_default</selectedoff>
          <selectedon>:/downloadscontrol/reload_default</selectedon>:/downloadscontrol/reload_default</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MouseHoverButton" name="btnUp">
        <property name="toolTip">
         <string>Move up</string>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/downloadscontrol/move_up_default</normaloff>
          <normalon>:/downloadscontrol/move_up_clicked</normalon>
          <disabledoff>:/downloadscontrol/move_up_disabled</disabledoff>
          <disabledon>:/downloadscontrol/move_up_disabled</disabledon>
          <activeoff>:/downloadscontrol/move_up_hover</activeoff>
          <activeon>:/downloadscontrol/move_up_hover</activeon>
          <selectedoff>:/downloadscontrol/move_up_default</selectedoff>
          <selectedon>:/downloadscontrol/move_up_default</selectedon>:/downloadscontrol/move_up_default</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MouseHoverButton" name="btnDown">
        <property name="toolTip">
         <string>Move down</string>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/resources.qrc">
          <normaloff>:/downloadscontrol/move_down_default</normaloff>
          <normalon>:/downloadscontrol/move_down_clicked</normalon>
          <disabledoff>:/downloadscontrol/move_down_disabled</disabledoff>
          <disabledon>:/downloadscontrol/move_down_disabled</disabledon>
          <activeoff>:/downloadscontrol/move_down_hover</activeoff>
          <activeon>:/downloadscontrol/move_down_hover</activeon>
          <selectedoff>:/downloadscontrol/move_down_default</selectedoff>
          <selectedon>:/downloadscontrol/move_down_default</selectedon>:/downloadscontrol/move_down_default</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="lineTrafficLimit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>2</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbTrafficLimit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enable traffic limit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbTrafficLimit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Traffic limit, kbps</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MouseHoverButton</class>
   <extends>QToolButton</extends>
   <header>mousehoverbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
